src/tests/test_utils/functional_test_utils/src/summary/op_summary.cpp

```diff
@@ -253,13 +253,13 @@ void OpSummary::saveReport() {
         auto processId = std::to_string(getpid());
         filename += "_" + processId + "_" + ts;
     }
-    filename += CommonTestUtils::REPORT_EXTENSION;
+    filename += ov::test::utils::REPORT_EXTENSION;
 
-    if (!CommonTestUtils::directoryExists(outputFolder)) {
-        CommonTestUtils::createDirectoryRecursive(outputFolder);
+    if (!ov::test::utils::directoryExists(outputFolder)) {
+        ov::test::utils::createDirectoryRecursive(outputFolder);
     }
 
-    std::string outputFilePath = outputFolder + std::string(CommonTestUtils::FileSeparator) + filename;
+    std::string outputFilePath = outputFolder + std::string(ov::test::utils::FileSeparator) + filename;
```

Comment:
@iefode @ilyachur @ilya-lavrenov @olpipi 
What do you think if we define some alias like `using TU = ov::test::utils` in order to remove extra symbols and simplify any next refactoring?
First of all, I'm concerned that 1/4 of symbols are namespaces instead of meaningful code

Reply:
I am against the global `using`, because it can introduce collisions.

In local spaces developers can use local `using` to make shorter some expressions.
In case if user uses some expression only one time, he shouldn't introduce new usings.